### PR TITLE
Remove right sidebar on home page 

### PR DESCRIPTION
**Pull request by annielovesjs**
- removed the right panel on home page
- test by launching the extension after building to check if its only on the home page where the panel doesnt show up

### CODE REVIEW

**Review by jasmingoh — APPROVED** [review]
Looks good!
image

File: src/client/src/containers/RightSidebar/styles.module.css [hunk 1]
```diff
@@ -12,6 +12,10 @@
   overflow: auto;
 }
 
+.rightViewCroppedSummaryPage {
+  width: 400px !important; /*change this width after top nav bar migration to change the width of the right side bar on summary page*/
+}
+
```
Comment (streetcoder123) [inline src/client/src/containers/RightSidebar/styles.module.css:18]: Did you mean to include style changes as part of this PR? I thought this PR is just about removing the instance of right sidebar from the home page.
Is this including the change for `greater width` of right side-bar on summary page?

**Comment by streetcoder123** [comment]
This PR contains all the changes from the PR #804 as well (to resize the right sidebar to have greater width on the summary page). **Hence this PR should only be merged after #804 is APPROVED AND MERGED.**

**Comment by annielovesjs** [comment]
closing this until the top nav bar PR is merged

**Pull request closed without merging**